src/sentry/integrations/pipeline.py

```diff
@@ -60,7 +60,8 @@ def _finish_pipeline(self, data):
             )
         else:
             integration = ensure_integration(self.provider.key, data)
-        integration.add_organization(self.organization.id)
+
+        org_integration = integration.add_organization(self.organization.id)
```

Comment:
this is serializes to a superset of the information?

Reply:
Yeah, if you look at the `OrganizationIntegration` serializer it actually serializes the entire Integration + configuration.

https://github.com/getsentry/sentry/blob/ba397b68677bf36088702e2adc6cb8c3c60f9f18/src/sentry/api/serializers/models/integration.py#L50-L82